entity-service/src/test/kotlin/com/egm/stellio/entity/repository/Neo4jRepositoryTests.kt

```diff
@@ -333,6 +333,25 @@ class Neo4jRepositoryTests {
         neo4jRepository.deleteEntity(entity.id)
     }
 
+    @Test
+    fun `it should return entities matching given type and ids`() {
```

Comment:
can you also write another test with a query passing a list of ids (and not only one)?